src/zope/sqlalchemy/datamanager.py

```diff
@@ -37,23 +36,27 @@
 except ImportError:
     pass
 else:
-    _retryable_errors.append((cx_Oracle.DatabaseError, lambda e: e.args[0].code == 8177))
+    _retryable_errors.append(
+        (cx_Oracle.DatabaseError, lambda e: e.args[0].code == 8177)
+    )
 
 # 1213: Deadlock found when trying to get lock; try restarting transaction
 try:
     import pymysql
 except ImportError:
     pass
 else:
-    _retryable_errors.append((pymysql.err.OperationalError, lambda e: e.args[0] == 1213))
+    _retryable_errors.append(
+        (pymysql.err.OperationalError, lambda e: e.args[0] == 1213)
+    )
 
 # The status of the session is stored on the connection info
-STATUS_ACTIVE = 'active'  # session joined to transaction, writes allowed.
-STATUS_CHANGED = 'changed'  # data has been written
-STATUS_READONLY = 'readonly'  # session joined to transaction, no writes allowed.
+STATUS_ACTIVE = "active"  # session joined to transaction, writes allowed.
+STATUS_CHANGED = "changed"  # data has been written
+STATUS_READONLY = "readonly"  # session joined to transaction, no writes allowed.
 STATUS_INVALIDATED = STATUS_CHANGED  # BBB
 
-NO_SAVEPOINT_SUPPORT = set(['sqlite'])
+NO_SAVEPOINT_SUPPORT = set(["sqlite"])
```

Comment:
This could be spelled `{"sqlite"}` nowadays, set literals are accepted since Python 2.7.